modules/microshift-audit-logs-config-proc.adoc

```diff
@@ -0,0 +1,66 @@
+// Text snippet included in the following assemblies:
+//
+// * microshift_configuring/microshift-audit-logs-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-audit-log-values_{context}"]
+= Configuring audit log values
+
+Use the {microshift-short} service configuration file to implement custom audit log settings.
```

Comment:
```suggestion
By using the {microshift-short} service configuration file, you can implement custom settings for the audit log.
```

Reply:
Better: You can configure audit log settings by using the {microshift-short} service configuration file.